### ISSUE TITLE
Different strip text sizes for nested facet

### ISSUE THREAD

**Issue by cliffbueno** [teunbrand/ggh4x#175]
Hello,
I love ggh4x, thanks for a great package!
I was wondering if it's possible to have different strip text sizes for the different nested variables.
For example, if I have  `facet_nested_wrap(~ var1 + var2)`, how can I make the top level (var1) have strip text size 14 and the bottom level (var2) have strip text size 12? I am aware of how `theme(strip.text = element_text(size = 12))` works, but that changes all of the strips together. We need either vectorized input to `element_text` or separate `strip.text` arguments for each.
Thanks,
Cliff

**Comment by teunbrand** [comment]
Hi Cliff,
Thanks for the commendations!
Yes that is definitely possible by using `strip = strip_nested(text_x = elem_list_text(size = c(14, 12)))`.
Hope that is sufficient to get you on the right track

**Comment by cliffbueno** [comment]
Thank you! So you already had it! This works.

Note to others, this is set in the `facet_nested_wrap()` function, not the `theme()` function, which is part of why I couldn't find the solution as I was looking for theme and strip.text options.

And instead of just 2 values, one for each variable, I specified a longer list of each individual value for each individual strip. So my variable 1 had 7 strips and my variable 2 had 10 strips so I ended up with
 `facet_nested_wrap(~ var1 + var2, strip = strip_nested(text_x = elem_list_text(size = c(14, 14, 14, 14, 14, 14, 14, 12, 12, 12, 12, 12, 12, 12, 12, 12, 12))))`
